src/AccessibilityInsights.VersionSwitcher/Properties/Resources.resx

```diff
@@ -117,14 +117,30 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadCommandLineMessageFormat" xml:space="preserve">
+    <value>Invalid Command line: {0}</value>
```

Comment:
nit: weird capitalization, should be `Invalid command line:`

Reply:
Fixed